src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -95,6 +96,34 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   return capture_result;
 }
 
+[[nodiscard]] static std::vector<ApiFunction> FindApiFunctions(
+    const orbit_client_data::ModuleManager& module_manager) {
+  // Api functions are declared in Orbit.h. They are implemented in user code through the
+  // ORBIT_API_INSTANTIATE macro. Those functions are used to query the tracee for Orbit specific
+  // information such as the memory location where Orbit should write function pointers to enable
+  // the Api after having injected liborbit.so.
+
+  std::vector<ApiFunction> api_functions;
+  for (const ModuleData* module_data : module_manager.GetAllModuleData()) {
+    constexpr const char* kOrbitApiGetAddressPrefix = "orbit_api_get_function_table_address_v";
+    for (size_t i = 0; i <= kOrbitApiVersion; ++i) {
+      std::string orbit_api_function_name = absl::StrFormat("%s%u", kOrbitApiGetAddressPrefix, i);
+      uint64_t function_hash = function_utils::GetHash(orbit_api_function_name);
+
+      const FunctionInfo* function_info = module_data->FindFunctionFromHash(function_hash);
```

Comment:
maybe add FindFunctionByName (you can use hash internally for now but at least do the verification for cases of hash collision)

Reply:
Yeah good idea, thanks, I did just that.